moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -409,6 +417,12 @@ std::optional<ServoParameters> ServoParameters::validate(ServoParameters paramet
                         "greater than zero. Check yaml file.");
     return std::nullopt;
   }
+  if (parameters.target_pose_lookahead_time < 0 || parameters.target_pose_lookahead_time > 0.1)
+  {
+    RCLCPP_WARN(LOGGER,
+                "Parameter 'target_pose_lookahead_time' should be greater than zero and typically less than 0.1s.");
```

Comment:
I would consider putting this to the config files instead as a comment for the parameter. That way, if someone truly wants to use some high value, there wouldn't be warnings but in all likelihood the user still knows that it is not a great idea.